src/grid_map_geo.cpp

```diff
@@ -85,6 +85,7 @@ bool GridMapGeo::initializeFromGeotiff(const std::string &path) {
 
   const OGRSpatialReference *spatial_ref = dataset->GetSpatialRef();
   std::string name_coordinate = spatial_ref->GetAttrValue("geogcs");
```

Comment:
Note - GetSpacialRef can return null, so the code would segfault from a nullptr dereference if the user supplied the path to a dataset that didn't have one.
https://gdal.org/doxygen/classGDALDataset.html#a9bd7f8b54ff4c12756976c7a0116d49a